SETUP_GUIDE.md

```
@@ -0,0 +1,477 @@
+7. 【部署】
+   /deploy-checklist
+   pnpm run deploy:preview
+   pnpm run deploy:cf
+
+8. 【清理】
+   /clear (为下一个任务清空历史)
+```
+
+### 提问方式优化
+
+**❌ 不高效的问法**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)